### PR TITLE
Issue 832: Start rest server in standalone gradle task

### LINKED ISSUE #832: Single node doesn't start rest server
**Problem description**
SingleNode gradle task doesn't start a rest server.

**Problem location**
SingleNode gradle task.

**Suggestions for an improvement**
Resolve conflicts with Jersey versions and start REST server in SingleNode task.

### PR DESCRIPTION
**Pull request by Abhijeet-Jadhav**
**Change log description**
Starting a rest server along with other Pravega components in standalone gradle task.

**Purpose of the change**
Fixes #832 

**What the code does**
Excludes com.sun.jersey dependency. 

**How to verify it**
./gradlew startStandalone 
Use curl/rest client to call rest apis on localhost:9091
eg.
`curl -i -X GET http://localhost:9091/ping`
```
HTTP/1.1 200 OK
content-length: 0
connection: keep-alive
```

### CODE REVIEW

File: standalone/src/main/java/io/pravega/local/InProcPravegaCluster.java [hunk 1]
```diff
@@ -302,6 +304,8 @@ private ControllerServiceMain startLocalController(int controllerId) {
                 .publishedRPCPort(this.controllerPorts[controllerId])
                 .build();
 
+        RESTServerConfig restServerConfig = RESTServerConfigImpl.builder().host("localhost").port(9091).build();
```
Comment (skrishnappa) [inline standalone/src/main/java/io/pravega/local/InProcPravegaCluster.java]: Don't hard code this port. Make it similar to how the RPC ports are set.

File: build.gradle [hunk 2]
```diff
@@ -404,6 +404,11 @@ project('standalone') {
         testCompile project(path:':segmentstore:server:host', configuration:'testRuntime')
     }
 
+    configurations {
+        runtime.exclude group: "com.sun.jersey", module: "jersey-core"
```
Comment (skrishnappa) [inline build.gradle]: I am assuming this does not impact the HDFS instance and related functionality that is being run as part of the standalone task. Ensure this is tested.

File: standalone/src/main/java/io/pravega/local/InProcPravegaCluster.java (hunk 1 above)
Comment (Abhijeet-Jadhav) [inline standalone/src/main/java/io/pravega/local/InProcPravegaCluster.java]: Done.

File: build.gradle (hunk 2 above)
Comment (Abhijeet-Jadhav) [inline build.gradle]: Standalone uses in memory storage. Does not deploy HDFS. 
cc @arvindkandhare

**Pull request merged**